### PR TITLE
update to april release

### PR DESCRIPTION
**Pull request by ltalirz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ltalirz** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
Any specific reason to choose the April release rather than the June one?

**Comment by ltalirz** [comment]
> Any specific reason to choose the April release rather than the June one?

The feature we needed was in the April release

**Comment by ltalirz** [comment]
Please have a look at https://github.com/conda-forge/lammps-feedstock/pull/201/commits/333d57daf4eb2db26fa2e62bd1136e4a1c845754

I gobbled this together from resources on the linter warnings but I can't claim I really understand all of it

File: recipe/meta.yaml [hunk 1]
```diff
@@ -54,10 +54,10 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - {{ stdlib('c') }}
```
Comment (jan-janssen) [inline recipe/meta.yaml:57]: We tried this in https://github.com/conda-forge/lammps-feedstock/pull/198 but that was not successful so far, so for the moment we just ignore the recommendation.

**Comment by ltalirz** [comment]
hm... getting this on osx
```
 nothing provides requested c_osx-64
```

File: recipe/meta.yaml (hunk 1 above)
Comment (ltalirz) [reply recipe/meta.yaml:57]: ok, let me try removing it again

**Comment by ltalirz** [comment]
@jan-janssen , so if the checks except for the linter were to pass, would you merge this?

Or is there something that still does not look right to you?

**Comment by ltalirz** [comment]
E.g. do I still need to add
```
os_version:
  # cos7 to get c_stdlib_version 2.17
  # which isn't working from conda_forge_config.yaml
  linux_64: cos7
```
to the  `conda-forge.yaml`?

**Comment by jan-janssen** [comment]
> @jan-janssen , so if the checks except for the linter were to pass, would you merge this?

I can merge it once the tests pass - no need to do anything more from your side.

**Pull request merged**